content/codeofconduct.md

```diff
@@ -73,7 +71,7 @@ All reports of breaches of the code of conduct will be investigated and handled
 
 ## The current PyMCon Diversity Committee consists of:
```

Comment:
All the "persons responsible for resolving complaints" is gone, and only this is left, formatted as a heading. If we only use numfocus form, then we don't need this section, _nor the conflict of interest one_ as they are not pymcon specific.

Similar comment applies to the timeline summary though. It most probably updated to reflect what will really happen. Now it doesn't reallly make any sense, there is no diversity committee, I am also not sure how much info will be available to us for example, will it be possible to make the summary of reported incidents?